homeassistant/components/rainbird.py

```diff
@@ -0,0 +1,105 @@
+"""
+Module for interacting with WiFi LNK module of the Rainbird Irrigation system.
+
+This project has no affiliation with Rainbird. This module works with the
+Rainbird LNK WiFi Module. For more information see:
+http://www.rainbird.com/landscape/products/controllers/LNK-WiFi.htm
+
+This module communicates directly towards the IP Address of the WiFi module it
+does not support the cloud. You can start/stop the irrigation and get the
+currenltly active zone.
+
+I'm not a Python developer, so sorry for the bad code. I've developed it to
+control it from my domtica systems.
+"""
+
+import logging
+import homeassistant.helpers as helpers
+
+REQUIREMENTS = ['pyrainbird==0.0.7']
+
+# Home Assistant Setup
+DOMAIN = 'rainbird'
+SERVER = ''
+PASSWORD = ''
+STATE_VAR = 'rainbird.activestation'
+
+_LOGGER = logging.getLogger(__name__)
+
+

```

Comment:
Configuration variables needs to run through the validation:

https://home-assistant.io/developers/code_review_component/#configuration